service/k8sclient/k8sclient_test.go

```diff
@@ -63,6 +63,37 @@ func TestK8sClient(t *testing.T) {
 
 	l := logger.Get(ctx)
 
+	t.Run("Install operators", func(t *testing.T) {
+		err = client.InstallXtraDBOperator(ctx)
+		require.NoError(t, err)
+
+		err = client.InstallPSMDBOperator(ctx)
+		require.NoError(t, err)
+
+		for i := 0; i < 5; i++ {
+			_, err = client.kubeCtl.Run(ctx, []string{"wait", "--for=condition=Available", "deployment", "percona-xtradb-cluster-operator"}, nil)
```

Comment:
do we have some timeouts for this operation? Can it hang forever?

Reply:
Default timeout is 30s.